plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -238,6 +239,18 @@ public PinotConfig setGrpcEnabled(boolean grpcEnabled)
         return this;
     }
 
+    public boolean isSecureConnection()
+    {
+        return secureConnection;
+    }
+
+    @Config("pinot.secure-connection")
```

Comment:
Please add an assertion to verify no conflict with `pinot.controller-urls`. For instance, when this config property is true, `pinot.controller-urls` shouldn't have `https` url.